app/experimenter/nimbus-ui/src/components/PageEditMetrics/index.tsx

```diff
@@ -5,11 +5,38 @@
 import React from "react";
 import { RouteComponentProps } from "@reach/router";
 import AppLayoutWithExperiment from "../AppLayoutWithExperiment";
+import FormMetrics from "../FormMetrics";
+import LinkExternal from "../LinkExternal";
+import { useConfig } from "../../hooks/useConfig";
 
 const PageEditMetrics: React.FunctionComponent<RouteComponentProps> = () => {
+  const { probeSets } = useConfig();
+
   return (
     <AppLayoutWithExperiment title="Metrics" testId="PageEditMetrics">
-      {({ experiment }) => <p>{experiment.name}</p>}
+      {({ experiment }) => (
+        <>
+          <p>
+            Every experiment analysis automatically includes how your experiment
+            has impacted <strong>Retention, Search Count, and Ad Click</strong>{" "}
+            metrics. Get more information on{" "}
+            {/* TODO Requires url https://jira.mozilla.com/browse/EXP-656 */}.
```

Comment:
. 😛
![Screen Shot 2020-12-03 at 4 04 03 PM](https://user-images.githubusercontent.com/6392049/101088139-34ee0700-3581-11eb-83da-3c287ae528c4.png)

Reply:
Oh I thought I fixed that

Reply:
Oh I fixed it in the other pr